MaxText/checkpointing.py

```diff
@@ -64,6 +72,53 @@ def create_orbax_checkpoint_manager(
   return mngr
 
 
+def create_orbax_emergency_checkpoint_manager(
```

Comment:
Is It necessary to have two separate creations for emergency and non-emergency? E.g. we can't just add the emergency checkpoint options (when requested) to the existing manager?

Reply:
The creation signatures of the emergency vs. regular checkpoint managers are very different, as evidenced by the parameter list. I am trying to keep the logic different instead of mixing the two.